### ISSUE TITLE
load_table() - Only creating table and script - Not populating the table

### ISSUE THREAD

**Issue by ayuba7272** [mark-hoffmann/fastteradata#13]
I'm trying to fast load a pandas dataframe from Python to a Teradata table. The ftd.load_table() function is creating the table with required column but the values are not getting populated from the csv to Teradata table. Is there something which I'm missing??

**Comment by jeuell** [comment]
I have this same problem.  Did you ever figure it out?

**Comment by jeuell** [comment]
When I run the fastload script separately, it gives me the following:
Warning: EOF on INPUT stream
for some reason, the fastload utility thinks the input file has and EOF right away but I cannot find anything suspicious in the csv file.

**Comment by ayuba7272** [comment]
> I have this same problem. Did you ever figure it out?

Nope. I finally ended up using other methods. Currently writing it to a Hadoop server and using swoop to tranfer the table to teradata

**Comment by jeuell** [comment]
I intercepted the fastload script and ran it manually and there was a column width issue I cannot explain.  I got it to load the table after some interventions but the module would need to be updated to expand the columns by default instead of using what comes out of the dataframe.  A couple of other issues I encountered were the interpretation of the instructions for the "env" argument.  The file "home" was not clear on my windows environment and the module couldn't find it so I changed it to point to the same directory as my load and script files.  when it says to replace everything between the double underscores in the .fastteradata credential file, it actually means replace everything between the double quotes including replacing the double underscores.  I am probably just a noob and don't know the common programming practices so it took me a couple of tries to figure that out.  Overall, this is an interesting package and kudos to the efforts by Mark Hoffman.  I don't think it is a simple plug and play package yet based on my experience.
